FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/ConfigurationResolvers/ConfigurationResolverV1.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations.Arrow;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Configurations.ConfigurationResolvers
+{
+    public class ConfigurationResolverV1
```

Comment:
Would it be possible that we put version number as parameter? If so, `ConfigurationResolver` would be responsible to resolve all versions of configs.